src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/setting/handler/PerformanceAnalyzerClusterSettingHandler.java

```diff
@@ -209,4 +211,14 @@ private int setBit(int number, int bitPosition) {
     private int resetBit(int number, int bitPosition) {
         return bitPosition < MAX_ALLOWED_BIT_POS ? (number & ~(1 << bitPosition)) : number;
     }
+
+    /**
+    * Checks if the pa is set or not.
+    *
+    * @param number The number which needs to be checked.
+    * @return true if the PA_ENABLED bit is set, false otherwise.
+    */
+    private boolean checkPerformanceAnalyzerEnabled(int number) {
+        return ((number & (1 << PA_ENABLED_BIT_POS)) == ENABLED_VALUE);
+    }
```

Comment:
Can we make this a little more generic like `checkBit(int clusterSettingValue)`?

Reply:
modified. it takes in bit position as well and method can be used to check if the bit is set or not.